live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommand.java

```diff
@@ -54,8 +54,6 @@ public record IngestMetadataCommand(
         ensureNonNull(name, "Node %s name delta cannot be null", nodeId);
         ensureNonNull(primaryAssocQName, "Node %s qualified name delta cannot be null", nodeId);
         ensureNonNull(nodeType, "Node %s type delta cannot be null", nodeId);
-        ensureNonNull(createdByUserWithId, "Node %s created by user with nodeId delta cannot be null", nodeId);
-        ensureNonNull(modifiedByUserWithId, "Node %s modified by user with nodeId delta cannot be null", nodeId);
```

Comment:
it actually checks if `propertyDelta` is null not its value or not so it should remain unchanged